tests/qtest/vhost-user-test.c

```diff
@@ -1156,3 +1158,33 @@ static void register_vhost_gpio_test(void)
                  "vhost-user-gpio", test_read_guest_mem, &opts);
 }
 libqos_init(register_vhost_gpio_test);
+
+static uint64_t vu_video_get_features(TestServer *s)
+{
+    return 0x1ULL << VHOST_USER_F_PROTOCOL_FEATURES;
+}
+
+static struct vhost_user_ops g_vu_video_ops = {
+    .type = VHOST_USER_VIDEO,
+
```

Comment:
Empty lines on purpose?

Reply:
Yes, other `vhost_user_ops` declarations in the same file have the same empty lines.